src/objective-c/tests/InteropTests/InteropTestsMultipleChannels.m

```diff
@@ -86,8 +86,9 @@ - (void)setUp {
   self.continueAfterFailure = NO;
 
   _remoteService = [RMTTestService serviceWithHost:kRemoteSSLHost callOptions:nil];
-
+#ifdef GRPC_COMPILE_WITH_CRONET
```

Comment:
This should always be tested with Cronet.

Reply:
Yeah thanks for pointing that out. That was for silencing Bazel's error messages. Now that we are not going to use Bazel for this test we can remove this.

Reply:
sg